packages/lib/versionInfo.ts

```diff
@@ -1,6 +1,24 @@
 import { _ } from './locale';
 import Setting from './models/Setting';
 import { reg } from './registry';
+import PluginService, { Plugins } from '@joplin/lib/services/plugins/PluginService';
+
+export type PluginInfo = {
+	name: string;
+	version: string;
+};
+
+// Get Plugins in Array Form
+export const getPluginsArr = (pluginsObj: Plugins): PluginInfo[] => {
+	const plugins: PluginInfo[] = [];
+	for (const pluginId in pluginsObj) {
+		plugins.push({
+			name: pluginsObj[pluginId].manifest.name,
+			version: pluginsObj[pluginId].manifest.version,
+		});
+	}
+	return plugins;
+};
```

Comment:
I don't understand the purpose of this function. You're looping on the array to create a new array. Then later you loop on this new array to create the plugin strings. So why not eliminate this intermediate step and loop on the first array to directly create the plugin strings?